src/createStore.js

```diff
@@ -1,7 +1,7 @@
-import $$observable from 'symbol-observable'
+import $$observable from '../node_modules/symbol-observable/es/index.js'
```

Comment:
This doesn't work on the web.

Reply:
why not?

symbol-observable 1.1.0 is compatible

Reply:
There isn't a `node_modules` folder on the web.

Reply:
Yes there is. This is how this feature is supposed to work.
You are supposed to upload node_modules that are imported.

This is the entire idea and reason this works in the first place.

Take a look at polymer, lodash etc. They follow the same patttern. 

I think you might have missed this feature in the latest browser releases

* https://hackernoon.com/native-es-modules-ready-for-prime-time-87c64d294d3c
* https://jakearchibald.com/2017/es-modules-in-browsers/
  * https://jakearchibald.com/2017/es-modules-in-browsers/#comment-3334824431
* more needed?

Reply:
No, that makes the assumption you're using npm or nodejs, both of which don't exist in the browser context. If I were to manually download the code here, independent of npm, it wouldn't work with this form. 

You're also dependant on the exact filesystem layout of the symbol-observable module. They can change that at any time (we're doing the same thing in redux 4.0), so you're relying on a private API essentially.

This is what I was getting at in the last issue where this stuff isn't ready for prime time.

Reply:
Thank you for the explanation. At least I understand your concerns now.

If users download the code here they'd probably use the dist version, right?  
How would it work without npm if they download it? 

The PR simply adds file extensions and the full path to node_modules. It doesn't even cause a breaking change IMHO.

Changes in symbol-observable should not matter since they use semver and the version is fixed in the package.json. I don't understand that concern at all, sorry. If you deliver the wrong version then it's not an issue with the package/module but a mistake the developer made. 

I see that I made an error with the relative path to `node_modules`. It cannot work like this  but I think if the path were absolute `/node_modules` which is the same in every npm project it should work.

I've tested it in a project using polymer, polymer-redux and redux, The setup is straightforward and everything is vanilla, no issues whatsoever. I'll gladly test it in one of my react projects built with webpack and babel too.

I also am willing to provide several test cases or forks of well known products built on redux to prove the compatibility. 

I'd just really love to see this feature in the near future.

Thank you for understanding.